android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/alipay/AlipaySocialSession.kt

```diff
@@ -0,0 +1,67 @@
+// Alipay SDK: https://opendocs.alipay.com/open/218/sxc60m
+package io.logto.sdk.android.auth.social.alipay
+
+import android.app.Activity
+import android.net.Uri
+import com.alipay.sdk.app.OpenAuthTask
+import io.logto.sdk.android.auth.social.SocialSession
+import io.logto.sdk.android.completion.Completion
+import io.logto.sdk.android.exception.LogtoException
+import io.logto.sdk.core.util.GenerateUtils
+
+class AlipaySocialSession(
+    override val context: Activity,
+    override val redirectTo: String,
+    override val callbackUri: String,
+    override val completion: Completion<String>,
+) : SocialSession {
+    companion object {
+        private const val APP_ID = "app_id"
```

Comment:
i don't think we need to define this since:
1. it varies by social identity provider
2. we only use the key once

Reply:
updated